FlowerySnails/Yak-Social-Network
# JCfeed

Description of Proposed Changes
The user can now post their thoughts for the YAK world to see. The posts push to our yak.json file and our app will pull those and print them to the DOM.

Steps to Test
Outline the steps to test

1. git fetch --all
2. git checkout JCfeed
3. code .
4. Run npm start in terminal
5. look at our webpage and see if the text box pops up under posts
6. Type post and either hit enter or post button
7. Make sure the post writes to the DOM and saves in our yak.json 
8. Tell Jake he did a great job 

Impacted Areas in Application
List general components of the application that this PR will affect:

templet.js
post.js
feed.js
yak.json


Definition of Done
The project must be fully documented. This includes the following:
Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
Every class must be documented with purpose, author, and methods.
Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
The project must be able to run fully, and without errors, on each teammate's system.
Fulfills every requirement.
Every line of code has been peer reviewed.
For projects that require unit testing, core functionality must be identified and have at least one test for each.
